<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1891</width>
    <height>957</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 5px</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1891</width>
     <height>961</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 20, 52);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>840</y>
      <width>1481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border:5px solid;
border-color: rgb(23, 20, 52);
border-bottom-color: rgb(199, 199, 199);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1780</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 202, 202);
font: 10pt &quot;Candara&quot;;</string>
    </property>
    <property name="text">
     <string>LogOut</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="chatBox">
    <property name="geometry">
     <rect>
      <x>393</x>
      <y>60</y>
      <width>1481</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(64, 56, 147, 0.4);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 75 16pt &quot;Candara&quot;;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>121</y>
      <width>321</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(64, 72, 81, 0);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>59</y>
      <width>351</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(64, 72, 81, 0.5);
border-radius: 10px;
color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="user">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
font: 20pt &quot;Cambria&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:None;</string>
     </property>
     <property name="text">
      <string>GRUPOS</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>321</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(64, 72, 81, 0);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonGroup">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>460</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Candara</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: None;</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>chatBox</zorder>
   <zorder>listWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget_2</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>699</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>901</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>1130</x>
     <y>880</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
